Apps/Sandcastle/gallery/Imagery Layers.html

```diff
@@ -28,22 +30,23 @@
     'use strict';
 //Sandcastle_Begin
 var viewer = new Cesium.Viewer('cesiumContainer', {
-    imageryProvider : new Cesium.ArcGisMapServerImageryProvider({
-        url : 'https://services.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer'
+    imageryProvider : Cesium.createWorldImagery({
+        style : Cesium.IonWorldImageryStyle.AERIAL_WITH_LABELS
     }),
     baseLayerPicker : false
 });
 
-var layers = viewer.imageryLayers;
-var earthAtNight = layers.addImageryProvider(new Cesium.IonImageryProvider({ assetId: 3812 }));
-earthAtNight.alpha = 0.5;
-earthAtNight.brightness = 2.0;
+var layers = viewer.scene.imageryLayers;
+var blackMarble = layers.addImageryProvider(new Cesium.IonImageryProvider({ assetId: 3812 }));
+
+blackMarble.alpha = 0.5; // 0.0 is transparent.  1.0 is opaque.
+
+blackMarble.brightness = 2.0; // > 1.0 increases brightness.  < 1.0 decreases.
```

Comment:
Picky... but maybe a little too concise. 
`Values range from 0.0 to 1.0, where 0 is transparent and 1.0 is opaque.`?
`Values > 1.0 increase brightness and values < 1.0 decrease.`?

Reply:
Actually, I think I should remove these comments entirely.